docs/ref/models/options.txt

```diff
@@ -288,6 +288,10 @@ Django quotes column and table names behind the scenes.
     incurs a cost to your database. Each foreign key you add will
     implicitly include all of its default orderings as well.
 
+    If a query doesn't have an ordering specified, results are returned from
+    the database in an unspecified order. A particular ordering is only
+    guaranteed if some ordering is specified.
```

Comment:
Thanks, I'll suggest this wording then: A particular ordering is guaranteed only when ordering by a set of fields that uniquely identify each object in the results. For example, if a ``name`` field isn't unique, ordering by it won't guarantee objects with the same name always appear in the same order.